data/transactions/logic/README.md

```diff
@@ -139,8 +139,6 @@ For three-argument ops, `A` is the element two below the top, `B` is the penulti
 | `getbyte` | pop a byte-array A and integer B. Extract the Bth byte of A and push it as an integer |
 | `setbyte` | pop a byte-array A, integer B, and small integer C (between 0..255). Set the Bth byte of A to C, and push the result |
 | `concat` | pop two byte-arrays A and B and join them, push the result |
-| `substring s e` | pop a byte-array A. For immediate values in 0..255 S and E: extract a range of bytes from A starting at S up to but not including E, push the substring result. If E < S, or either is larger than the array length, the program fails |
```

Comment:
Why are these gone?  I thought maybe they got re-ordered, but then wouldn't I see them being added elsewhere?

Reply:
I think it's because I moved the `substring` ops from the original arithmetic opgroup... I'll take a closer look at this too